cibuildwheel/__main__.py

```diff
@@ -308,7 +308,7 @@ def print_build_identifiers(
     elif platform == 'windows':
         python_configurations = cibuildwheel.windows.get_python_configurations(build_selector, architectures)
     elif platform == 'macos':
-        python_configurations = cibuildwheel.macos.get_python_configurations(build_selector)
+        python_configurations = cibuildwheel.macos.get_python_configurations(build_selector, architectures)
```

Comment:
We _could_ just use `module =getattr(cibuildwheel, platform)`, then `python_configurations = platform.get_python_configurations(build_selector, architectures)` now that they all match, and drop the whole if, if you wanted. (or some more static version of that with a dict or simpler if).

Reply:
I think it's mostly good fortune that they match right now, so I think I'd rather spell it out. Plus mypy probably groks it better this way.